examples/pipelines/hf_dataset_pipeline/components/load_from_hub/build_image.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Read component configs
+. ../../components.config
+
+# Set some variables
+ARTIFACT_PATH=${PROJECT_ARTIFACT_PATH}
+IMAGE_NAME="components/${PWD##*/}"
+IMAGE_TAG="latest"
+
+# Create full name of the image
+FULL_IMAGE_NAME=${ARTIFACT_PATH}/${IMAGE_NAME}:${IMAGE_TAG}
+echo $FULL_IMAGE_NAME
+
```

Comment:
I think you can include here `gcloud config set builds/use_kaniko True` to enable it automatically and avoid having the user look through the docs for figuring out this step.
 Can you please include it in all the build_image.sh that exist? you can also update the docs then to just mention that cashing is used

Reply:
Would propose not to do this in this PR and move to `docker` instead of `gcloud` anyway.

Reply:
Will keep this for now, and in a follow-up PR we can update all components to leverage `docker` (in a single script that builds the images for all components)